Video-Transformers/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Video Transformers Changelog
+
+All notable changes to this project will be documented in this file.
+
+## 2.25.2
+
+### Added
+
+- Support pre-built transformers in the Vonage Media Processor library or create your own custom video transformer to apply to published video.
+
+### Known issues
+
+- When using Vonage's Background Blur, the ML model is not being automatically added to the main bundle. It is required to be added manually: 
```

Comment:
Still needed? I thought you fixed this.

Reply:
It is fixed and already merged but it will only be available on 2.25.3